train.lua

```diff
@@ -16,6 +16,7 @@ cmd:option('--minLR', 0.00001, 'minimum learning rate')
 cmd:option('--saturateEpoch', 20, 'epoch at which linear decayed LR will reach minLR')
 cmd:option('--maxEpoch', 50, 'maximum number of epochs to run')
 cmd:option('--batchSize', 10, 'mini-batch size')
+cmd:option('--gpu', 0, 'Zero-indexed ID of the GPU to use; for CPU mode set --gpu = -1')
```

Comment:
Why set to `-1` w/ CPU? Isn't the value ignored?

Also, why use a zero-indexed ID if torch starts at one?

I'm not familiar w/ multi GPU support, so sorry if I'm missing something...